src/app/loginComponents/accounts/controls/controls.component.html

```diff
@@ -11,7 +11,7 @@ <h3>Change your Username</h3>
   <mat-card>
     <h3>Delete your Dockstore Account</h3>
     <p>This action is only available for accounts with no published tools/workflows and have not shared workflows via permissions.</p>
-    <button class="mr-1 button" mat-raised-button (click)="deleteAccount()" [disabled]="(canChangeUsername$ | async) === false">
```

Comment:
the .button background color is identical to `color="warn"`, the only difference is that the color was changed to white which I applied globally